frontend/lib/pages/loc-confirmation.tsx

```diff
@@ -12,13 +13,80 @@ import { EmailAttachmentForm } from '../email-attachment';
+
+  const [isExpanded, toggleExpansion] = useState(false);
+
+  return (
+    <div className="jf-sanitation-guidelines notification is-warning">
+      <div>
+        Please be aware that letting a repair-worker into your home to make repairs may expose you to the Covid-19 virus. 
+        In order to follow social distancing guidelines and to limit your exposure, please follow these steps to stay as safe as possible.
+        {' '}<button className={classnames("button","is-text","is-paddingless","is-uppercase", isExpanded && "is-hidden")} 
+              onClick={() => toggleExpansion(true)}>
+          Learn More
+        </button>
+      </div>
+      <div className={classnames("content", !isExpanded && "is-hidden")}>
+        <div className="columns">
+          <div className="column is-one-quarter">
+            <SquareImage size={128} src='frontend/img/sanitation-guide/chat.svg' alt="" />
+          </div>
+          <div className="column">
+            <h3 className="is-size-6 is-uppercase has-text-weight-bold has-text-grey-dark">Before the repair-worker arrives</h3>
+            <p>Talk to anyone that you live with and let them know that a repair-worker is coming to perform the repairs that you requested.</p>
+          </div>
+        </div>
+        <div className="columns">
+          <div className="column is-one-quarter">
+            <SquareImage size={128} src='frontend/img/sanitation-guide/hands.svg' alt="" />
+          </div>
+          <div className="column">
+            <h3 className="is-size-6 is-uppercase has-text-weight-bold has-text-grey-dark">While the repair-worker is inside your home</h3>
+            <p>Have the repair-worker wash their hands with soap for at least 20 seconds as soon as they come into your house.</p>
+            <p>If possible, stay in a different room from where the work is being done. If a separate room is not available, maintain at least a six-foot distance from the repair-worker until the repair is completed.</p>
+          </div>
+        </div>
+        <div className="columns">
+          <div className="column is-one-quarter">
+            <SquareImage size={128} src='frontend/img/sanitation-guide/wipe.svg' alt="" />
+          </div>
+          <div className="column">
+            <h3 className="is-size-6 is-uppercase has-text-weight-bold has-text-grey-dark">After the repair-worker leaves</h3>
+            <p>Immediately sanitize all surfaces in your home, especially doorknobs, the sink where the repair-worker washed their hands, and any surfaces you know they have likely been in contact with.</p>
+            <p className="is-size-7">For guidance on how to thoroughly sanitize your home and a list of recommended effective cleaning products visit 
+              {' '}<OutboundLink href="https://www.cdc.gov/coronavirus/2019-ncov/prepare/cleaning-disinfection.html" target="_blank">Center for Disease Control (CDC) Guide on How to Clean and Disinfect</OutboundLink>
```

Comment:
So ideally we should be adding this from the `OutboundLink` component, but it doesn't seem to be adding it itself... we should ideally be putting a `rel="noopener noreferrer"` on there... blech.  Hopefully the CDC won't ever get compromised though.

Reply:
Oh, I assumed OutboundLink included the `rel="noopener noreferrer"` prop! Maybe something we can add later?